docs/components/modeler/desktop-modeler/troubleshooting.md

```diff
@@ -0,0 +1,110 @@
+---
+id: troubleshooting
+title: Troubleshooting
+description: "This page lists common issues with Desktop Modeler and potential resolutions."
+---
+
+## How to start Desktop Modeler without plugins
+
+You can start Desktop Modeler with the [`--disable-plugins` flag](./flags/flags.md#disable-plug-ins).
```

Comment:
Adjusted "the modeler" to Desktop Modeler -- is this what you meant?

Reply:
:+1: